jobs/ssh/regenerate_puppet_certificate.erb

```diff
@@ -0,0 +1,25 @@
+<%#
+name: Regenerate puppet certificate
+job_category: Puppet
+description_format: Regenerate puppet certificate
+snippet: false
+provider_type: SSH
+kind: job_template
+%>
+
+<% # this template requires safe mode to be disabled -%>
+
+<%= render_template('Service Action - SSH Default', :service => 'puppet', :action => 'stop') -%>
+
+
+<% if preview? -%>
+#   this would delete a certificate on puppet ca, but running in preview mode
+<% else -%>
+  <%= @host.puppet_ca_proxy.statuses[:puppetca].destroy(@host.name) && '# deleted cert on proxy' %>
+<% end -%>
+# delete the old cert and key
+rm -rf /var/lib/puppet/ssl/*
+# generate a new one
+puppet agent --onetime --no-usecacheonfailure --verbose --tags no_such_tag
```

Comment:
the question is if we want to create autosign entry manually or let user manually verify the request in UI, maybe that could be user input of the template?